vocabulary-draft/source-code-version-control.md

```diff
@@ -0,0 +1,36 @@
+# Source Code Version Control Events
+
+This phase includes events related to changes in Source Code repositories that are relevant from a Continuous Delivery perspective.
+
+
+## Repository Events
+
+These events are related to Source Code repositories
+- **Repository Created Event**: a new Source Code Repository was created to host source code for a project
+- **Repository Modified Event**: a Source Code Repository modified some of its attributes, like location, or owner
```

Comment:
I think there is a risk of confusion between the modification of repository "information" (e.g. location or owner) and modification of the repository "content", e.g. new branches or commits.

I propose we rename this event to either `repository metadata updated` or `repository information updated`

Reply:
@erkist so to be clear, you think that the name of the event needs to be more clear. In my perspective, the description is stating that already.  I wonder if for example the repository URL is considered metadata, as where the repository represents the repository itself. I am happy to change the name of the event if that makes it more clear.

Reply:
@salaboy I agree that the description makes it clear, but would prefer if it was clear already by the name of the event.
And maybe metadata is the wrong term? I'm looking for a term that indicates that it is not the content of the repository that has been modified (see for instance https://git-scm.com/book/en/v2/Git-Basics-Recording-Changes-to-the-Repository which uses the term modification to talk about the content of the repo).

But I don't have a strong opinion on it.